perllib/FixMyStreet/Cobrand/UK.pm

```diff
@@ -449,4 +452,50 @@ sub check_recaptcha {
         unless $res->{success};
 }
 
+sub public_holidays {
+    my $nation = shift || 'england-and-wales';
+    my $json = _get_bank_holiday_json();
+    return [ map { $_->{date} } @{$json->{$nation}{events}} ];
+}
+
+sub is_public_holiday {
+    my %args = @_;
+    $args{date} ||= localtime;
+    $args{date} = $args{date}->date;
```

Comment:
Does localtime return an object?

Reply:
Yes, if you "use Time::Piece" :)
(The commits are re-ordered, I think github displays them in the wrong order though.)